src/libraries/Microsoft.PowerFx.Interpreter/Functions/Library.cs

```diff
@@ -44,27 +43,17 @@ private static T GetService<T>(this IServiceProvider services, T defaultService)
 
         public static readonly IReadOnlyDictionary<TexlFunction, AsyncFunctionPtr> FunctionImplementations;
 
+        internal static readonly TexlFunctionSet<TexlFunctionImplementation> TexlFunctionImplementations = new TexlFunctionSet<TexlFunctionImplementation>();
```

Comment:
>static readonly TexlFunctionSet [](http://example.com/codeflow?start=17&length=31)

threading warning because this is static mutable -

Reply:
Get rid of TexlFunctionImplementation 
- just used locally within library.cs cctor. 


Revert Library.cs

- fixes threading warning 

that removes all usage of TexlFunctionImplementation 

which means we don't need to add interface ITexlFunction 
Which means TexlFunctionSet isn't generic